### PR TITLE
port of Sandy's examples from Keybow 2040 library

### PR DESCRIPTION
**Pull request by titimoby**
Thanks for the library

### CODE REVIEW

File: examples/hid-keys-simple.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+# A simple example of how to set up a keymap and HID keyboard on Keybow 2040.
+
+# You'll need to connect Keybow 2040 to a computer, as you would with a regular
+# USB keyboard.
+
+# Drop the rgbkeypad.py file into your `lib` folder on your `CIRCUITPY` drive.
+
+# NOTE! Requires the adafruit_hid CircuitPython library also!
+
+import board
```
Comment (AngainorDev) [inline examples/hid-keys-simple.py]: No more needed

File: examples/hid-keys-simple.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+#
+# SPDX-License-Identifier: MIT
+
+# Original file was done for Keybow 2040
+# Author: Sandy Macdonald
+
+# Adaptation for RGB Keypad
+# Author: Thierry Chantier
+
+# A simple example of how to set up a keymap and HID keyboard on Keybow 2040.
```
Comment (AngainorDev) [inline examples/hid-keys-simple.py]: Update and simplify comments, to remove confusing Keybow 2040 references

File: examples/hid-keys-simple.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+# You'll need to connect Keybow 2040 to a computer, as you would with a regular
+# USB keyboard.
+
+# Drop the rgbkeypad.py file into your `lib` folder on your `CIRCUITPY` drive.
+
+# NOTE! Requires the adafruit_hid CircuitPython library also!
+
+import board
+from rgbkeypad import RgbKeypad, number_to_xy, hsv_to_rgb
```
Comment (AngainorDev) [inline examples/hid-keys-simple.py]: number_to_xy, hsv_to_rgb are not used

File: examples/hid-keys-simple.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        keyboard.send(keycode)
+        key.set_led(*rgb)
+
+    # A release handler that turns off the LED
+    @keypad.on_release(key)
+    def release_handler(key):
+        key.led_off()
+
+while True:
+    # Always remember to call keybow.update()
```
Comment (AngainorDev) [inline examples/hid-keys-simple.py]: keypad

**Review by AngainorDev — COMMENTED** [review]
Thanks for the contribution!
Left a few comments, mostly unused imports and references to keybow in comments.

File: examples/hid-keys-simple.py (hunk 2 above)
Comment (titimoby) [reply examples/hid-keys-simple.py]: I understand but I also wanted to be super clear that I had minimal changes to do on the original work and credit properly Sandy

Comment (AngainorDev) [reply examples/hid-keys-simple.py]: Of course. 
Not speaking of the credits and licence header, but specifically the description of the script
"A simple example of how to set up a keymap and HID keyboard on Keybow 2040."
=> "A simple example of how to set up a keymap and HID keyboard on the Pimoroni RGB Keypad"
and maybe remove the extra directions 
"You'll need to connect Keybow 2040 to a computer, as you would with a regular USB keyboard."
if not, edit the device name as well.

**Comment by titimoby** [comment]
I think I've cleaned up

**Comment by AngainorDev** [comment]
Thanks!

**Pull request merged**